src/cloudfront-control-plane-stack.ts

```diff
@@ -141,7 +141,8 @@ export class CloudFrontControlPlaneStack extends Stack {
     uri.startsWith('/project') || 
     uri.startsWith('/pipelines') || 
     uri.startsWith('/plugins') || 
-    uri.startsWith('/alarms') || 
+    uri.startsWith('/alarms') ||  
+    uri.startsWith('/analytics') || 
```

Comment:
Should we add it to `rewrite rules` in CloudFront?

Reply:
We using this CloudFront function for URL rewriting.